trunk/SUAVE/Methods/Noise/Fidelity_One/Engine/mixed_noise_component.py

```diff
@@ -15,8 +15,37 @@
 # ---------------------------------------------------------------------- 
 
 ## @ingroupMethods-Noise-Fidelity_One-Engine
-def mixed_noise_component (SPL_m,Velocity_primary,theta_m,sound_ambient,Velocity_secondary,Velocity_aircraft,Area_primary,Area_secondary,DSPL_m,EX_m,Str_m,Velocity_mixed,XBPR):
-    """This function calculates the noise contribution of the mixed jet component"""
+def mixed_noise_component(SPL_m,Velocity_primary,theta_m,sound_ambient,Velocity_secondary,
+                          Velocity_aircraft,Area_primary,Area_secondary,DSPL_m,EX_m,Str_m,Velocity_mixed,XBPR):
+    """This function calculates the noise contribution of the mixed jet component
+    
+    Assumptions:
+        N/A
+
+    Source:
+        None
```

Comment:
Source should be referenced

Reply:
I am not sure what is the source. This is Ebraer's code